### ISSUE TITLE
Wizard previous/next buttons are not focusable

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#368]
On the wizard, the previous and next buttons are now not focusable and can't be used with a keyboard.

![screen shot 2017-11-06 at 10 38 35](https://user-images.githubusercontent.com/1682452/32434807-6a22f75e-c2df-11e7-8135-03aa1d782d21.png)

Checking the wizard on the plugin current version, these buttons have a tabindex=0.
However, in the yoast-components, they get a tabindex=-1.

I guess something has changed in `material-ui` and to my understanding, `disableKeyboardFocus` was previously used just for styling purposes. Instead, now it is used also to control the buttons focusability. See https://github.com/Yoast/yoast-components/pull/70 where `disableKeyboardFocus` was used for styling purposes.

I think we should just remove `disableKeyboardFocus: true`.

Aside: as far as I see, `material-ui` does lots of things with focusability of elements, often overriding the native behavior and rebuilding it programmatically. I'm not sure I like so much this approach :(
